Bottom: a6c1485a9eb0f35596d70c474545cd09b2ac1ced
Top:    50bc424cce708c3d8239e083ed2dd66684df260d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:57:50 +0000

Remove debug from outline generation


---

diff --git a/src/polygon.c b/src/polygon.c
index 9fe8cf6..57cfe80 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -105,7 +105,7 @@ dicer output is used for HIDs which cannot render things with holes
 #endif
 
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
 
@@ -2220,18 +2220,19 @@ struct clip_outline_info {
 
 #define ROUTER_THICKNESS MIL_TO_COORD (10)
 //#define ROUTER_THICKNESS MIL_TO_COORD (0.1)
-
+#undef ROUTER_THICKNESS
+#define ROUTER_THICKNESS (line->Thickness)
 static int
 arc_outline_callback (const BoxType * b, void *cl)
 {
-  ArcType *arc = (ArcType *)b;
+  ArcType *line = (ArcType *)b;
   struct clip_outline_info *info = cl;
   POLYAREA *np, *res;
 
 #ifdef DEBUG_CIRCSEGS
-  if (!(np = ArcPoly (arc, arc->Thickness)))
+  if (!(np = ArcPoly (line, line->Thickness)))
 #else
-  if (!(np = ArcPoly (arc, ROUTER_THICKNESS)))
+  if (!(np = ArcPoly (line, ROUTER_THICKNESS)))
 #endif
     return 0;
 
diff --git a/src/polygon1.c b/src/polygon1.c
index 5461e2b..6fc0f81 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -115,7 +115,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 //#define DEBUG_JUMP
 //#define DEBUG_GATHER
 #undef DEBUG_ANGLE
-//#define DEBUG
+#undef DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
